resources/prometheus/prometheus-rules.yaml

```diff
@@ -539,11 +539,24 @@ spec:
             )
           record: central:success_rate10m
 
+        - expr: |
+            probe_success{app="rhacs", endpoint="central-ui"}
```

Comment:
I assume this is not related to the probe service? Could you please add a comment what is probed here?

Reply:
Yes, it's exposed by the blackbox-exporter. I've added a comment.